core/trino-main/src/main/java/io/trino/server/security/KerberosConfig.java

```diff
@@ -79,6 +81,19 @@ public KerberosConfig setKeytab(File keytab)
         return this;
     }
 
+    @FileExists
+    public File getKeyCache()
+    {
+        return keyCache;
+    }
+
+    @Config("http-server.authentication.krb5.keycache")
+    public KerberosConfig setKeyCache(File keyCache)
+    {
+        this.keyCache = keyCache;
+        return this;
+    }
+
```

Comment:
Can we apply some sort of a validation here so that we could either keyTab or credentialCache but not both, like how we do in hive connector.